code/modules/antagonists/heretic/knowledge/ash_lore.dm

```diff
@@ -44,7 +44,7 @@
 		/obj/item/match = 1,
 	)
 	result_atoms = list(/obj/item/melee/sickly_blade/ash)
-	limit = 2
+	limit = 20 //SKYRAT EDIT - ORIGINAL: 2
```

Comment:
5 at most, so it's not something they can afford to keep throwing away

Reply:
Sure, I'll go with that for now.